sources/packages/api/src/services/user/user.service.ts

```diff
@@ -14,4 +14,18 @@ export class UserService extends DataModelService<User> {
       userName,
     });
   }
+
+  async updateUserStatus(userId: number, isActive: boolean){
+    return this.repo
+    .createQueryBuilder()
+    .update(User)
+    .set({ isActive: isActive})
+    .where("id = :id", { id: userId })
+    .execute();
+  }
+
+  async getActiveUser(userName: string) {
+    return this.repo.findOne({ where: {userName:userName, isActive: true} });
```

Comment:
I believe that the line could be simplified as below. If you think that it is better, please adjust it.

```
return this.repo.findOne({userName:userName, isActive: true});
```